en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -6,190 +6,154 @@ iconBrand: php
 
 # PHP Coding Conventions
 
-This document provides guidelines for code formatting and documentation
-to developers contributing to SilverStripe. It applies to all PHP files
-in the `framework/` and `cms/` modules, as well as any supported additional modules.
+This document provides guidelines for code formatting and documentation to developers contributing to Silverstripe. It applies to all PHP files in supported modules, as well as any additional modules under the `silverstripe` GitHub account.
 
-Coding standards are an important aspect for every software project,
-and facilitate collaboration by making code more consistent and readable.
+Coding standards are an important aspect for every software project, and facilitate collaboration by making code more consistent and readable.
```

Comment:
```suggestion
```
This is a very generic statement that should probably go in the generic [coding conventions](https://docs.silverstripe.org/en/5/contributing/coding_conventions/) page.

Reply:
I simply removed it